website/docs/reference/network.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Network
```

Comment:
Maybe this should be "Network view"? Wdyt? Just "Network" doesn't really tell you all that much. Network overview? 

```suggestion
title: Network view
```

Reply:
Network view is fine, it's an overarching term over "Network overview" (which is one tab) and "Network traffic"